simplified-app-ekirjasto/src/main/assets/txnative/en/txstrings.json

```diff
@@ -591,6 +864,9 @@
       "catalogOperationFailed" : {
          "string" : "The operation could not be completed."
       },
+      "catalogPlaceholder" : {
+         "string" : "Lorem ipsum dolor sit amet, probatus volutpat has at, vis adhuc iuvaret omittantur an, sadipscing conclusionemque his ad. Eam ut simul tempor. Primis consetetur appellantur vim eu, eum an iudico dolorum. Ad vis utamur honestatis, sanctus debitis referrentur an eam."
```

Comment:
This should be removed from all languages in Transifex, the original Android string has been removed, but it still exists in Transifex.

Reply:
Cannot be removed through the Transifex UI due to license restrictions, but should be possible to remove via the API, I'll handle these separate of this PR.